--- geany-plugins-1.38/git-changebar/src/gcb-plugin.c.orig	2021-09-19 15:29:36.000000000 +0200
+++ geany-plugins-1.38/git-changebar/src/gcb-plugin.c	2022-06-29 06:26:19.499733381 +0200
@@ -211,6 +211,7 @@ static const struct {
 };
 
 
+#if ! defined (LIBGIT2_VER_MINOR) || (LIBGIT2_VER_MAJOR == 0) || ((LIBGIT2_VER_MAJOR == 1) && (LIBGIT2_VER_MINOR < 4))
 /* workaround https://github.com/libgit2/libgit2/pull/3187 */
 static int
 gcb_git_buf_grow (git_buf  *buf,
@@ -227,6 +228,7 @@ gcb_git_buf_grow (git_buf  *buf,
   return git_buf_grow (buf, target_size);
 }
 #define git_buf_grow gcb_git_buf_grow
+#endif
 
 static void
 buf_zero (git_buf *buf)
@@ -234,7 +236,9 @@ buf_zero (git_buf *buf)
   if (buf) {
     buf->ptr = NULL;
     buf->size = 0;
+#if ! defined (LIBGIT2_VER_MINOR) || (LIBGIT2_VER_MAJOR == 0) || ((LIBGIT2_VER_MAJOR == 1) && (LIBGIT2_VER_MINOR < 4))
     buf->asize = 0;
+#endif
   }
 }
 
