diff -ur geany-plugins-1.36/git-changebar/src/gcb-plugin.c geany-plugins-1.36-libgit1/git-changebar/src/gcb-plugin.c
--- geany-plugins-1.36/git-changebar/src/gcb-plugin.c	2019-03-24 18:58:57.000000000 +0100
+++ geany-plugins-1.36-libgit1/git-changebar/src/gcb-plugin.c	2020-05-17 01:08:19.618324822 +0200
@@ -32,11 +32,11 @@
 #include <geany.h>
 #include <document.h>
 
-#if ! defined (LIBGIT2_SOVERSION) || LIBGIT2_SOVERSION < 22
+#if (LIBGIT2_VER_MAJOR == 0) && (LIBGIT2_VER_MINOR < 22)
 # define git_libgit2_init     git_threads_init
 # define git_libgit2_shutdown git_threads_shutdown
 #endif
-#if ! defined (LIBGIT2_SOVERSION) || LIBGIT2_SOVERSION < 23
+#if (LIBGIT2_VER_MAJOR == 0) && (LIBGIT2_VER_MINOR < 23)
 /* 0.23 added @p binary_cb */
 # define git_diff_buffers(old_buffer, old_len, old_as_path, \
                           new_buffer, new_len, new_as_path, options, \
@@ -45,7 +45,7 @@
                     new_buffer, new_len, new_as_path, options, \
                     file_cb, hunk_cb, line_cb, payload)
 #endif
-#if ! defined (LIBGIT2_SOVERSION) || LIBGIT2_SOVERSION < 28
+#if (LIBGIT2_VER_MAJOR == 0) && (LIBGIT2_VER_MINOR < 28)
 # define git_buf_dispose  git_buf_free
 # define git_error_last   giterr_last
 #endif
diff -ur geany-plugins-1.36/workbench/src/plugin_main.c geany-plugins-1.36-libgit1/workbench/src/plugin_main.c
--- geany-plugins-1.36/workbench/src/plugin_main.c	2019-07-25 19:11:40.000000000 +0200
+++ geany-plugins-1.36-libgit1/workbench/src/plugin_main.c	2020-05-17 01:08:47.295824244 +0200
@@ -36,7 +36,7 @@
 #include "tm_control.h"
 
 
-#if ! defined (LIBGIT2_SOVERSION) || LIBGIT2_SOVERSION < 22
+#if (LIBGIT2_VER_MAJOR == 0) && (LIBGIT2_VER_MINOR < 22)
 # define git_libgit2_init     git_threads_init
 # define git_libgit2_shutdown git_threads_shutdown
 #endif
